voter/src/test/java/org/terracotta/voter/ActiveVoterTest.java

```diff
@@ -278,7 +280,7 @@ private void disconnectManagers(Collection<ClientVoterManager> managers) throws
         Thread.sleep(500);
         return false;
       });
-      when(manager.registerVoter(VOTER_ID)).thenReturn(-1L);
+      Mockito.doReturn(1L).when(manager).registerVoter(VOTER_ID);
```

Comment:
negative value makes the disconnection intention clear.

Reply:
this was a transcription error.  doReturn is used because it works better with spy